tests/mocks/databasemock.js

```diff
@@ -23,7 +23,7 @@
 	});
 
 	var dbType = nconf.get('database'),
-		testDbConfig = nconf.get('test_database'),
+		testDbConfig = nconf.get('test_database_'+dbType),
```

Comment:
Any reason why this needs to be set? We'd only run tests against one db set at a time anyway, so `test_database` is sufficient.